src/docs/resources/videos.md

```diff
@@ -14,101 +14,127 @@ but there are many others.
 
 ## Series
 
-The following list of series features the introduction to the series,
-with a link to the complete playlist.
+Here are some of the series offered on the
+[flutterdev YouTube channel][].
 
-### Flutter Interact 2019
+[flutterdev YouTube channel]: {{site.youtube-site}}/channel/UCwXdFgeE9KYzlDdR7TG9cMw
 
-Flutter Interact was a day-long event that showcased
-the latest from Flutter, and focused on creating beautiful
-experiences across devices.
+### Flutter Engage 2021
 
-Check out the Flutter Interact 2019 Highlights Reel:
+[Flutter Engage 2021][] was a day-long event that
+officially launched Flutter 2.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/EgBMGDtHZhE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+[Flutter Engage 2021]: https://events.flutter.dev/
 
-Watch recordings of the sessions on the Flutter Youtube channel.
+Check out the Flutter Engage 2021 highlights reel:
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/wgTBLj7rMPM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-Vision Keynote<br>
-[Flutter Interact 2019 playlist][]
+<iframe width="560" height="315" src="{{site.youtube-site}}/embed/IdrCyS7EF8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-### Flutter in Focus
+Watch recordings of the sessions on the Flutter YouTube channel:
+
+<iframe width="560" height="315" src="{{site.youtube-site}}/embed/zSbsIiluixw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+Keynote<br>
+[Flutter Engage 2021 playlist][]
+
+Watch recordings of the sessions offered by the Flutter community:
+
+<iframe width="560" height="315" src="{{site.youtube-site}}/embed/-_de6IfY2RU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+[Flutter Engage community talks playlist][]
+
+[Flutter Engage 2021 playlist]: {{site.youtube-site}}/playlist?list=PLjxrf2q8roU21cXt24HLm-ZODXmr4Jw0C
+[Flutter Engage community talks playlist]: {{site.youtube-site}}/playlist?list=PLjxrf2q8roU1ln3WoiGTUvLr7ZrPPFvXv
+
+### Decoding Flutter
+
+Also known as "What the Flutter?" This series
```

Comment:
While I'd love to have this aka for the series, officially it's nothing other than "Decoding Flutter"